### ISSUE TITLE
Have you trained from scratch?

### ISSUE THREAD

**Issue by youansheng** [chenyuntc/simple-faster-rcnn-pytorch#64]
I have trained faster rcnn with this scripts. But the loss is higher than you report.
